contrib/asmap/README.md

```diff
@@ -9,4 +9,73 @@ Example usage:
 python3 asmap-tool.py encode /path/to/input.file /path/to/output.file
 python3 asmap-tool.py decode /path/to/input.file /path/to/output.file
 python3 asmap-tool.py diff /path/to/first.file /path/to/second.file
+python3 asmap-tool.py diff-addrs /path/to/first.file /path/to/second.file addrs.file
+```
+These commands may take a few minutes to run with `python3`,
+depending on the amount of data involved and your machine specs.
+Consider using `pypy3` for a faster run time.
+
+### Encoding and Decoding
+
+ASmap files are somewhat large (~30MB) in text form, and need to be encoded
```

Comment:
nit: Well, the ones we make with kartograf are that large. If someone else makes one based on their personal BGP dump or they don't use all the sources in kartograf they may be smaller. But it's not a big deal, the point is that people understand that we want to reduce the size. I would have just left the specific number out. But I would say you leave as is unless you have to retouch.

Reply:
yea fair -- mentioning the data size doesn't make sense. removed in [6afffba](https://github.com/bitcoin/bitcoin/pull/32110/commits/6afffba34e086de4cf0bb86729e12d116c1dcc9b)